Makefile

```diff
@@ -0,0 +1,3 @@
+lint:
+	@./node_modules/.bin/eslint .
```

Comment:
And tj himself seems like stop using makefile to run npm script in his recent (an year) repos 😢.